### ISSUE TITLE
Switching back and forth between p5 and JS

### ISSUE THREAD

**Issue by estherhersh** [peblio/proto-2#50]
The default text doesn't change back

**Comment by MathuraMG** [comment]
@estherhersh  - I was thinking about this and have a question
What do we want the user experience to be? When a user is say typing code under p5 and then realises its actually python code, and want to compile it under python. Then they switch to python - we wouldn't want the code to change right?

**Comment by estherhersh** [comment]
Yeah I think so. It’s an interesting question because  the hope is that the
user decides what language they want to use before they start coding. I’d
say the code should stay there if the language is changed  but we should
discuss it when Regina comes back. Maybe they should be prompted in some
way to pick a language and stick with it.

**Comment by MathuraMG** [comment]
@estherhersh - yae I would also think that the code stays. Let's leave this one open for now then
